apps/charterafrica/src/lib/data/common/processPageContributors.js

```diff
@@ -8,7 +8,7 @@ import formatDateTime from "@/charterafrica/utils/formatDate";
 import labelsPerLocale from "@/charterafrica/utils/translationConstants";
 
 const queryBuilder = (query) => {
-  const { search, location } = query;
+  const { search, location, classification } = query;
```

Comment:
The data team confirmed collection === classification?

Reply:
@kilemensi 

From this [document](https://docs.google.com/document/d/1HDQBaOJD8XjtJfo-RR5s-TOdh1njC88ZmDqUVNeJwxg/edit?usp=sharing),
<img width="779" alt="image" src="https://github.com/CodeForAfrica/ui/assets/39160236/49377c87-9d59-49b7-bbf3-4de211f7e8f5">

Reply:
Thanks @koechkevin ... If I understand the doc, `classification` is the _source_ for `collection`, it doesn't mean `classification` **is** `collection`. Similar to `DB` is the source for `Tool` but you can't say `DB` **is** `Tool`.

But anyways, let's get it over the line and push on academy.AFRICA.